src/read.fs

```diff
@@ -387,7 +391,7 @@ let readParameterDeclaration (checker: TypeChecker) (pd: ParameterDeclaration):
             | Some t -> readTypeNode checker t
             | None -> simpleType "obj"
     
-    let name = pd.name |> getBindingName
+    let name = pd.name |> getBindingName |> Option.defaultValue ("p" + string iParam)
```

Comment:
This is the relevant change: If the binding has no valid name (because it is a pattern), then just use "p%i"